src/main/java/org/folio/service/orders/PurchaseOrderLineService.java

```diff
@@ -424,29 +432,49 @@ public Future<String> updatePoLineReceiptStatus(PoLine poLine, PoLine.ReceiptSta
   }
 
   public Future<Void> validateAndNormalizeISBN(List<CompositePoLine> compositePoLines, RequestContext requestContext) {
+    return validateAndNormalizeISBNCommon(compositePoLines, requestContext, Predicates.alwaysFalse(),
+      UnaryOperator.identity(), ProductId::setProductId);
+  }
+
+  public Future<Void> validateAndNormalizeISBNAndProductType(List<CompositePoLine> compositePoLines, RequestContext requestContext) {
+    return inventoryCache.getInvalidISBNProductTypeId(requestContext)
```

Comment:
Why we need to invoke this method: getInvalidISBNProductTypeId() ?

Reply:
maybe it can be just a constant or inventory has separate product type with name: Invalid ISBN ?

Reply:
All right, the inventory has an identifier type called 'Invalid ISBN'.